quickstarts/fluentd-plugin-for-logs/config.yml

```diff
@@ -16,12 +16,11 @@ documentation:
     description: |
       Open source data collector, which lets you unify the data collection and
       consumption for a better use and understanding of data.
-    url: >-
-      https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIiwiZmF2b3JpdGVzIjp7InNlbGVjdGVkIjp0cnVlLCJ2aXNpYmxlIjp0cnVlfSwibGFzdFZpZXdlZCI6eyJzZWxlY3RlZCI6ZmFsc2UsInZpc2libGUiOnRydWV9fQ==&cards[0]=eyJuZXJkbGV0SWQiOiJpbnN0cnVtZW50LWV2ZXJ5dGhpbmctbG9ncy5zZXR1cC1mbHVlbnRkIiwiYWNjb3VudElkIjoyMTM2Njc5fQ==&sidebars[0]=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5jYXRlZ29yaWVzIiwicm9vdE5lcmRsZXRJZCI6Im5yMS1jb3JlLmxpc3RpbmciLCJmYXZvcml0ZXMiOnsic2VsZWN0ZWQiOnRydWUsInZpc2libGUiOnRydWV9LCJsYXN0Vmlld2VkIjp7InNlbGVjdGVkIjpmYWxzZSwidmlzaWJsZSI6dHJ1ZX19&state=f4a4ef04-fdff-7b52-4078-540018df0099
-
+    url: https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIiwiZmF2b3JpdGVzIjp7InNlbGVjdGVkIjp0cnVlLCJ2aXNpYmxlIjp0cnVlfSwibGFzdFZpZXdlZCI6eyJzZWxlY3RlZCI6ZmFsc2UsInZpc2libGUiOnRydWV9fQ==&cards[0]=eyJuZXJkbGV0SWQiOiJpbnN0cnVtZW50LWV2ZXJ5dGhpbmctbG9ncy5zZXR1cC1mbHVlbnRkIiwiYWNjb3VudElkIjoyMTM2Njc5fQ==&sidebars[0]=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5jYXRlZ29yaWVzIiwicm9vdE5lcmRsZXRJZCI6Im5yMS1jb3JlLmxpc3RpbmciLCJmYXZvcml0ZXMiOnsic2VsZWN0ZWQiOnRydWUsInZpc2libGUiOnRydWV9LCJsYXN0Vmlld2VkIjp7InNlbGVjdGVkIjpmYWxzZSwidmlzaWJsZSI6dHJ1ZX19&state=f4a4ef04-fdff-7b52-4078-540018df0099
```

Comment:
Lets consider not using a NR1 link here, this link will not work for the EU environment